sei_dbt/models/staging/stg_FCT_Health_client_202304.sql

```diff
@@ -3,4 +3,4 @@
 
 SELECT STUDY_ID,	BIRTH_YR_MON,	SEX,	POSTAL_CODE,	CITY,	STREET_LINE,	LHA,
 '2023-05-25' AS EFF_DATE,  '2023-05-25' as END_DATE
-FROM read_csv_auto("C:/Users/JDUAN/OneDrive - Government of BC/2024-025 Brett and Jon Database Test Warehouse/raw_data/CLR_EXT_20230525.csv")
+FROM read_csv_auto("raw_data/CLR_EXT_20230525.csv")
```

Comment:
Just leaving a note here that we'll have to figure out how to make these data paths actually work.

Reply:
Yes. After searching and try and error, I got it working with 'profiles.yml'